components/sidebar/index.jsx

```diff
@@ -15,12 +17,47 @@ const Sidebar = React.memo(({ className }) => {
           className="close"
         />
       </div>
-      <ul>
-        <li>Overview</li>
-        <li>Data</li>
-        <li>Statistics</li>
-        <li>Plugins</li>
-        <li>Settings</li>
+      <ul className="sidebar-navigation">
+        <li>
+          <Link href="/">
+            <a href="/">
+              <Icon iconType="overview" />
```

Comment:
I am not sure about having an explicit `Icon`. On the other hand we do not have any way to use icons in CSS. I will figure out something.